affinity/src/main/c/Makefile

```diff
@@ -10,6 +10,18 @@ TARGET := $(TARGET_DIR)/libCEInternals.so
 
 WORKING_DIR := $(TARGET_DIR)/../jni
 
+UNAME_S:= $(shell uname -s)
+ifeq ($(UNAME_S), Linux)
+    JNI_OS := linux
+    LRT := -lrt
+endif
+ifeq ($(UNAME_S), Darwin)
+    JNI_OS := darwin
+endif
+ifeq ($(UNAME_S), Windows)
+    JNI_OS := win32
```

Comment:
unfortunately the Windows build still failing in TC

Reply:
windows support now added, and TC green